--- plugins/javalib/org/munin/plugin/jmx/GCTime.java	2012-03-14 19:18:05.000000000 +0400
+++ plugins/javalib/org/munin/plugin/jmx/GCTime.java.new	2012-05-04 15:20:52.000000000 +0400
@@ -17,6 +17,6 @@
                         "CopyTime.label MinorTime\n" +
-			"CopyTime.type DERIVE\n" +
+			"CopyTime.type GAUGE\n" +
 			"CopyTime.info The approximate accumulated collection elapsed time in milliseconds. This method returns -1 if the collection elapsed time is undefined for this collector.\n" +
  		        "MarkSweepCompactTime.label MajorTime\n" +
- 		        "MarkSweepCompactTime.type DERIVE\n" +
+ 		        "MarkSweepCompactTime.type GAUGE\n" +
                         "MarkSweepCompactTime.info The approximate accumulated collection elapsed time in milliseconds. This method returns -1 if the collection elapsed time is undefined for this collector.The Java virtual machine implementation may use a high resolution timer to measure the elapsed time. This method may return the same value even if the collection count has been incremented if the collection elapsed time is very short. \n"
