diff -Nur munin-1.4.6.orig/dists/redhat/munin-node.rc munin-1.4.6/dists/redhat/munin-node.rc
--- munin-1.4.6.orig/dists/redhat/munin-node.rc	2011-07-05 11:36:14.000000000 -0600
+++ munin-1.4.6/dists/redhat/munin-node.rc	2011-08-27 16:35:55.056612853 -0600
@@ -16,6 +16,7 @@
 
 mkdir -p /var/run/munin 2>/dev/null
 chown munin /var/run/munin
+restorecon /var/run/munin 2>/dev/null
 
 # See how we were called.
 case "$1" in
