diff -ruN munin-1.4.5-orig/master/lib/Munin/Master/Config.pm munin-1.4.5/master/lib/Munin/Master/Config.pm
--- munin-1.4.5-orig/master/lib/Munin/Master/Config.pm	2010-02-09 06:45:00.000000000 -0600
+++ munin-1.4.5/master/lib/Munin/Master/Config.pm	2011-06-01 22:18:17.145761204 -0500
@@ -148,7 +148,7 @@
 		local_address    => 0,
 		logdir           => $Munin::Common::Defaults::MUNIN_LOGDIR,
 		max_processes    => $MAXINT,
-		rundir           => '/tmp',
+		rundir           => $Munin::Common::Defaults::MUNIN_STATEDIR,
 		timeout          => 180,
 		tls              => 'disabled',
 		tls_ca_certificate => "$Munin::Common::Defaults::MUNIN_CONFDIR/cacert.pem",
