diff -Nur munin-1.4.5.orig/node/lib/Munin/Node/Server.pm munin-1.4.5/node/lib/Munin/Node/Server.pm
--- munin-1.4.5.orig/node/lib/Munin/Node/Server.pm	2010-05-22 16:54:31.000000000 -0600
+++ munin-1.4.5/node/lib/Munin/Node/Server.pm	2011-02-07 15:15:51.527998974 -0700
@@ -171,7 +171,7 @@
 
     logger ("DEBUG: Running command \"$_\".") if $config->{DEBUG};
     if (/^list\s*([0-9a-zA-Z\.\-]+)?/i) {
-        _list_services($session, $1);
+        _list_services($session, lc($1));
     }
     elsif (/^cap\s?(.*)/i) {
         _negotiate_session_capabilities($session, $1);
@@ -309,7 +309,7 @@
 sub _list_services {
     my ($session, $node) = @_;
 
-    $node ||= $config->{fqdn};
+    $node ||= lc($config->{fqdn});
 
     if (exists $nodes{$node}) {
         my @services = @{$nodes{$node}};
