diff -Nur munin-1.2.5.orig/node/node.d.linux/fw_conntrack.in munin-1.2.5/node/node.d.linux/fw_conntrack.in
--- munin-1.2.5.orig/node/node.d.linux/fw_conntrack.in	2006-10-17 06:27:35.000000000 -0600
+++ munin-1.2.5/node/node.d.linux/fw_conntrack.in	2007-11-30 21:54:22.000000000 -0700
@@ -84,7 +84,7 @@
         exit 0
 	;;
     autoconf)
-        if [ -f /proc/net/ip_conntrack ] ; then
+        if [ -f /proc/net/ip_conntrack -o -f /proc/net/nf_conntrack ] ; then
 	    echo yes
 	    exit 0
 	else
@@ -102,7 +102,15 @@
 # udp      17 53 src=80.111.68.163 dst=62.179.100.29 sport=34153 dport=53 src=62.179.100.29 dst=80.111.68.163 sport=53 dport=34153 [ASSURED] use=1
 #
 
-cat /proc/net/ip_conntrack | awk '
+if [ -f /proc/net/ip_conntrack ]; then
+    conntrack_file=/proc/net/ip_conntrack
+elif [ -f /proc/net/nf_conntrack ]; then
+    conntrack_file=/proc/net/nf_conntrack
+else
+    exit 1
+fi
+
+cat "$conntrack_file" | awk '
   BEGIN  { STATE["ESTABLISHED"]=STATE["FIN_WAIT"]=STATE["TIME_WAIT"]=0;
 	   ASSURED=NOREPLY=NATED=STATE["SYN_SENT"]=STATE["UDP"]=0; }
   /^tcp/ { STATE[$4]++; }
diff -Nur munin-1.2.5.orig/node/node.d.linux/fw_forwarded_local.in munin-1.2.5/node/node.d.linux/fw_forwarded_local.in
--- munin-1.2.5.orig/node/node.d.linux/fw_forwarded_local.in	2006-10-17 06:27:35.000000000 -0600
+++ munin-1.2.5/node/node.d.linux/fw_forwarded_local.in	2007-11-30 21:57:02.000000000 -0700
@@ -36,10 +36,13 @@
 	if ( cat /proc/net/ip_conntrack 2>/dev/null >/dev/null ); then
 		echo yes
 		exit 0
+		elif ( cat /proc/net/nf_conntrack 2>/dev/null >/dev/null ); then
+			echo yes
+			exit 0
 	else
 		if [ $? -eq 127 ]
 		then
-			echo "no (ipconntrack not found)"
+			echo "no (ip_conntrack/nf_conntrack not found)"
 			exit 1
 		else
 			echo no
@@ -50,7 +53,7 @@
 
 if [ "$1" = "config" ]; then
 
-	echo 'graph_title ipconntrack'
+	echo 'graph_title conntrack'
 	echo 'graph_args -l 0 --base 1000'
 	echo 'graph_vlabel established connections'
 	echo 'graph_category network'
@@ -63,4 +66,13 @@
 	exit 0
 fi
 
-cat /proc/net/ip_conntrack | grep ESTABLISHED | sed 's/.*src=\(.*\) .*dst=\(.*\) sport.*src=\(.*\) .*dst=\(.*\) spo.*/\1 \2 \3 \4/g' | awk 'BEGIN {{FORW=0} {LOCAL=0}} {if($1 != $4){ FORW += 1} else { LOCAL += 1}} END {{print "forward.value "FORW} {print "local.value "LOCAL}}'
+if [ -f /proc/net/ip_conntrack ]; then
+	conntrack_file=/proc/net/ip_conntrack
+elif [ -f /proc/net/nf_conntrack ]; then
+	conntrack_file=/proc/net/nf_conntrack
+else
+	exit 1
+fi
+
+cat "$conntrack_file" | grep ESTABLISHED | sed 's/.*src=\(.*\) .*dst=\(.*\) sport.*src=\(.*\) .*dst=\(.*\) spo.*/\1 \2 \3 \4/g' | awk 'BEGIN {{FORW=0} {LOCAL=0}} {if($1 != $4){ FORW += 1} else { LOCAL += 1}} END {{print "forward.value "FORW} {print "local.value "LOCAL}}'
+
