diff -ur munin-1.4.5/plugins/lib/Munin/Plugin/SNMP.pm munin-1.4.5-fixed/plugins/lib/Munin/Plugin/SNMP.pm
--- munin-1.4.5/plugins/lib/Munin/Plugin/SNMP.pm	2010-05-05 19:02:43.000000000 +0200
+++ munin-1.4.5-fixed/plugins/lib/Munin/Plugin/SNMP.pm	2011-06-10 00:10:28.969751949 +0200
@@ -477,7 +477,7 @@
 
         my $response = $handle->get_request($oid);
 
-        if (!defined $response->{$oid} or $handle->error_status) {
+        if (!defined $response->{$oid} or $handle->error_status or $response->{$oid} eq 'noSuchObject') {
             print STDERR "# Error getting $oid: ",$handle->error(),"\n"
                 if $DEBUG;
             return;
